### PR TITLE
Efrs 1042

### PR DESCRIPTION
**Pull request by JSnik**

### CODE REVIEW

**Comment by JSnik** [comment]
I have read the CLA Document and I hereby sign the CLA

File: ui/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { RouterModule, Routes } from '@angular/router';
 import { MainLayoutComponent } from './ui/main-layout/main-layout.component';
 import { DemoLayoutComponent } from './ui/demo-layout/demo-layout.component';
 import { UserInfoResolver } from './core/user-info/user-info.resolver';
+import {ManageCollectionComponent} from "./ui/manage-collection/manage-collection.component";
```
Comment (isobolev) [inline ui/src/app/app-routing.module.ts:22]: please enable prettier in your IDE, and format all changed files.

File: ui/src/app/features/tool-bar/tool-bar.component.ts [hunk 2]
```diff
@@ -39,6 +39,10 @@ export class ToolBarComponent {
 
   constructor(private dialog: MatDialog, private translate: TranslateService) {}
 
+  ngOnInit(): void {
```
Comment (isobolev) [inline ui/src/app/features/tool-bar/tool-bar.component.ts:42]: please remove empty ngOnInit

File: ui/src/app/features/user-table/user-table.component.ts [hunk 3]
```diff
@@ -98,4 +101,14 @@ export class UserTableComponent extends TableComponent implements OnInit, OnChan
       this.message = this.createMessage;
     }
   }
+
+  refreshPage(): void {
+    if (!localStorage.getItem('foo')) {
```
Comment (isobolev) [inline ui/src/app/features/user-table/user-table.component.ts:106]: not very informative variable name. What are you trying to do here?

**Review by isobolev — COMMENTED** [review]
also please name name PR in the following manner: <ticket name> - <short description>

**Comment by JSnik** [comment]
i fixed them

File: ui/src/app/app.module.ts [hunk 4]
```diff
@@ -104,7 +104,9 @@ import { MatSelectModule } from '@angular/material/select';
     },
   ],
   bootstrap: [AppComponent],
-  exports: [],
+  exports: [
```
Comment (isobolev) [inline ui/src/app/app.module.ts:107]: please enable prettier in your IDE and format all files changed in this PR

File: ui/src/app/features/tool-bar/tool-bar.component.ts [hunk 5]
```diff
@@ -39,6 +39,8 @@ export class ToolBarComponent {
 
   constructor(private dialog: MatDialog, private translate: TranslateService) {}
 
+  ngOnInit(): void {}
```
Comment (isobolev) [inline ui/src/app/features/tool-bar/tool-bar.component.ts:42]: please remove empty `ngOnInit(): void {}` and `implements OnInit`

File: ui/src/app/features/user-list/user-list.component.ts [hunk 6]
```diff
@@ -27,7 +27,7 @@ import { ITableConfig } from '../table/table.component';
 import { UserListFacade } from './user-list-facade';
 
 @Component({
-  selector: 'app-user-list-container',
+  selector: ' app-user-list-container',
```
Comment (isobolev) [inline ui/src/app/features/user-list/user-list.component.ts:30]: format

File: ui/src/app/features/user-table/user-table.component.ts [hunk 7]
```diff
@@ -24,6 +24,7 @@ import { RoleEditDialogComponent } from '../role-edit-dialog/role-edit-dialog.co
 import { MatDialog } from '@angular/material/dialog';
 import { UserRole } from '../../data/interfaces/user-role';
 
+
```
Comment (isobolev) [inline ui/src/app/features/user-table/user-table.component.ts:27]: format

File: ui/src/app/features/user-table/user-table.component.ts [hunk 8]
```diff
@@ -98,4 +100,13 @@ export class UserTableComponent extends TableComponent implements OnInit, OnChan
       this.message = this.createMessage;
     }
   }
+
+  refreshPage(): void {
+    if (!localStorage.getItem('refresh-page')) {
+      localStorage.setItem('refresh-page', 'no reload');
```
Comment (isobolev) [inline ui/src/app/features/user-table/user-table.component.ts:106]: will this reload the page every time user is navigated to user table? There should be more appropriate fix than this

**Review by isobolev — COMMENTED** [review]
please rename your PR

File: ui/src/app/features/user-table/user-table.component.ts (hunk 8 above)
Comment (JSnik) [reply ui/src/app/features/user-table/user-table.component.ts:106]: okay i will fix them tonight

**Review by isobolev — APPROVED** [review]

**Pull request merged**